.github/workflows/doc-build.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       submodules: 'true'
       repository: pytorch/executorch
       upload-artifact: docs
+      timeout: 90
```

Comment:
Not related to MobileBert, but this timeout hasn't been set yet, so doc jobs timeout sometimes after 30m, i.e. https://github.com/pytorch/executorch/actions/runs/7663108516/job/20885389022?pr=1658.  Setting it to 90m like the rest of CI jobs.